### ISSUE TITLE
WebUI blank page on localhost

### ISSUE THREAD

**Issue by christophmayrhofer** [itsgoingd/clockwork#343]
I can load the web ui when it is deployed on my Azure WebApp.

However I get a blank page when I load /__clockwork on localhost.

The source code referencing the js/css files is there as expected, but there seems to be an issue when the js code is being executed:

```
bootstrap:2 Uncaught TypeError: Cannot read property '0' of undefined
    at e.value (bootstrap:2)
    at new e (bootstrap:2)
    at Module.56d7 (bootstrap:2)
    at i (bootstrap:2)
    at Object.0 (bootstrap:2)
    at i (bootstrap:2)
    at s (bootstrap:2)
    at bootstrap:2
    at bootstrap:2
```

So data seems to be undefined here:

```
function webpackJsonpCallback(data) {...}
```

**Comment by Tongzzzzz** [comment]
I get the same problem.

![微信截图_20190805152045](https://user-images.githubusercontent.com/26567782/62446198-b3237900-b794-11e9-891f-cdb28e5e709d.png)

**Comment by itsgoingd** [comment]
Hey, here is a Clockwork build with source maps, can you give it a try so we can get a more useful backtrace maybe? [clockwork-w-sourcemaps.zip](https://github.com/itsgoingd/clockwork/files/3469301/clockwork-w-sourcemaps.zip)

**Comment by spaceemotion** [comment]
Got the same error today. Here's the source mapped stacktrace:

```
standalone.js:88 Uncaught TypeError: Cannot read property '0' of undefined
    at Standalone.getCookie (standalone.js:88)
    at new Profiler (profiler.js:21)
    at Module../src/main.js (main.js:29)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
    at Object.0 (vendor.js:24)
    at __webpack_require__ (bootstrap:723)
    at bootstrap:790
    at bootstrap:790
```

**Comment by itsgoingd** [comment]
Thanks for looking into this, this is very helpful.

The original code used named group in the regex, which I removed because it wasn't supported in Firefox yet (https://github.com/underground-works/clockwork-app/commit/71506835c9589dab93b23e929cc05d5dc34ac2c7), but didn't update the second line correctly, which should be just `matches[1]`.

**Comment by itsgoingd** [comment]
Fixed in 4.0.7.
